pkg/k8s/service_cache.go

```diff
@@ -495,7 +495,7 @@ func (s *ServiceCache) correlateEndpoints(id ServiceID) (*Endpoints, bool) {
 		}
 	}
 
-	if svcFound && svc.IncludeExternal && svc.Shared {
```

Comment:
this is to revert back the changes in previous PR, the svc struct here is local service only.